README.md

```diff
@@ -1,10 +1,14 @@
 # FsWarn
-alpine based rootfs containing everything needed to warn the users that FsGuard failed the filesystem verification
 
-## Building:
-The build can only be done in alpine linux, a container can be used for this.
-```
+Alpine based rootfs containing everything needed to warn the users that FsGuard failed the filesystem verification.
+
+## Building
+
+The build for FsWarn can only be done in Alpine linux, a container can be used for this purpose.
```

Comment:
This would probably be better as "Building FsWarn is only possible on Alpine Linux, a container can be used for this purpose."